src/meepgeom.cpp

```diff
@@ -1978,8 +1978,8 @@ void fragment_stats::update_stats_from_material(material_type mat, size_t pixels
           nonzero_conductivity_pixels_extmat_already_added =
               count_nonzero_conductivity_pixels(med, pixels);
         }
-        break;
       }
+      break;
```

Comment:
Thanks!  Technically, the fall-through was safe since it just goes to the `return` statement, but the old code would cause a problem if we ever added more cases to this `switch` statement.

Reply:
Thanks for the feedback. Here the position change for the "break" is for a logic issue. The original version will only executes `i=0`, and then escape the `for` loop.

Reply:
Oh, I see, it is in a `loop`; I was looking at the wrong line, thanks.